src/app/profile/overview/overview.component.html

```diff
@@ -1,82 +1,208 @@
-<div class="container-fluid content padding-top-15">
-  <div class="row">
-    <div class="col-xs-12 col-md-2 col-md-offset-1">
-      <div>
+<f8-feature-toggle featureName="Profile.newOverview">
+  <div class="container-fluid content" user-level>
+    <div *ngIf="!context.user.attributes.emailVerified && viewingOwnAccount"
+      class="row email-alert padding-top-20">
+      <div class="col-xs-12 col-md-10 col-md-offset-1">
+        <div class="alert alert-warning">
+          <span class="pficon pficon-warning-triangle-o"></span>
+          <strong>Your email address is unverified.</strong> Please verify your email address by clicking the link we sent you.
+          <a href="javascript:void(0)" (click)="sendEmailVerificationLink()" class="alert-link pull-right">Resend verification link</a>
+        </div>
+      </div>
+    </div>
+    <div class="row profile-overview-masthead">
+      <div class="col-xs-4 col-md-1">
         <div *ngIf="(context.user.attributes.imageURL !== undefined && context.user.attributes.imageURL.length !== 0); then showImageURL else addImageURL"></div>
         <ng-template #showImageURL>
-          <img src="{{context.user.attributes.imageURL}}" class="img-responsive profile-img icon-user">
+          <img src="{{context.user.attributes.imageURL}}" class="img-responsive icon-user">
         </ng-template>
         <ng-template #addImageURL>
           <a href="javascript:void(0)"
+              *ngIf="viewingOwnAccount"
               [routerLink]="['/', context?.user?.attributes?.username, '_update']"
               [queryParams]="{ imageUrl: true }">
             <img src="../../../assets/images/profile-user.png" class="img-responsive profile-img-none">
             <span class="pficon pficon-edit edit-profile-img"></span>
           </a>
         </ng-template>
-        <h3 class="col-sm-12 profile-wrapper-username">{{context?.user?.attributes?.fullName}}</h3>
-        <p class="col-sm-12 profile-wrapper-username">
-          {{context?.user?.attributes?.username}}
+      </div>
+      <div class="col-xs-12 col-md-2">
+        <p class="profile-wrapper-name">{{context?.user?.attributes?.fullName}}
+          <span class="profile-wrapper-username">
```

Comment:
I'd recommend moving this outside of the `<p>` element. For one, I'm getting odd wrapping of the username, but `profile-wrapper-username` is also sized in proportion to the `<p>`. If you move this outside of that, then you can size it against the base site font instead.

<img width="497" alt="screen shot 2018-05-15 at 5 11 35 pm" src="https://user-images.githubusercontent.com/4032718/40083775-3454dd58-5863-11e8-83e2-6d9b3cbc8fcd.png">